kraken/managedcluster_scenarios/run.py

```diff
@@ -0,0 +1,68 @@
+import yaml
+import logging
+import sys
+import time
+from kraken.managedcluster_scenarios.managedcluster_scenarios import managedcluster_scenarios
+import kraken.managedcluster_scenarios.common_managedcluster_functions as common_managedcluster_functions
+import kraken.cerberus.setup as cerberus
+
+
+# Get the managedcluster scenarios object of specfied cloud type
+def get_managedcluster_scenario_object(managedcluster_scenario):
+    return managedcluster_scenarios()
+
+# Run defined scenarios
+def run(scenarios_list, config, wait_duration):
+    for managedcluster_scenario_config in scenarios_list:
+        with open(managedcluster_scenario_config, "r") as f:
+            managedcluster_scenario_config = yaml.full_load(f)
+            for managedcluster_scenario in managedcluster_scenario_config["managedcluster_scenarios"]:
+                managedcluster_scenario_object = get_managedcluster_scenario_object(managedcluster_scenario)
+                if managedcluster_scenario["actions"]:
+                    for action in managedcluster_scenario["actions"]:
+                        start_time = int(time.time())
+                        inject_managedcluster_scenario(action, managedcluster_scenario, managedcluster_scenario_object)
+                        logging.info("Waiting for the specified duration: %s" % (wait_duration))
+                        time.sleep(wait_duration)
+                        end_time = int(time.time())
+                        cerberus.get_status(config, start_time, end_time)
+                        logging.info("")
+
+
+# Inject the specified managedcluster scenario
+def inject_managedcluster_scenario(action, managedcluster_scenario, managedcluster_scenario_object):
+    generic_cloud_scenarios = ("stop_klusterlet_scenario", "managedcluster_crash_scenario")
```

Comment:
generic_cloud_scenarios not used.

Reply:
Just some minor nits on running flake8.  Thanks for the patch, will give it a try on a RHOSAK cluster :).